README.md

```diff
@@ -82,7 +101,8 @@ Describe the differences between unit and functional testing. What type of testi
 Your answer:
 ```text
 
-Replace this with your answer
+Unit testing is to check to see if methods are working and functional tests are to see if specific functionalty of an app is working usually on the client side. Rspec is a unit testing becuase it checks for methods.
+
 ```
```

Comment:
Functional testing doesn't necessarily mean we're testing something on the client-side. Functional testing means we are testing a specific functionality of an application, which may entail testing multiple methods at a time.

Unit testing can test methods, but it does not only test methods.